src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -2491,6 +2491,195 @@ def oil_water_surface_tension_model(p, t):
+            return s_lw_ref + p * 0
+
+        pressure_values = np.linspace(0.5, 1e10, 4)  # Pa (1e-5 to 1e5 in bar)
+        enthalpy_values = np.linspace(2617360.0, 2869860, 30)
+
+        temperature_values = temperature(pressure_values, enthalpy_values)
+
+        gas_density_model = (
+            ideal_gas_density_model if ideal_gas else constant_gas_density_model
+        )
+
+        t, p = np.meshgrid(temperature_values, pressure_values)
+        h, p = np.meshgrid(enthalpy_values, pressure_values)
+
+        data = [
+            gas_density_model(p, t).flatten(),
+            gas_density_derivative_respect_pressure(p, t).flatten(),
+            gas_density_derivative_respect_temperature(p, t).flatten(),
+            constant_density_model(p, t).flatten(),
+            oil_density_derivative_respect_pressure(p, t).flatten(),
+            oil_density_derivative_respect_temperature(p, t).flatten(),
+            gas_viscosity_model(p, t).flatten(),
+            oil_viscosity_model(p, t).flatten(),
+            surface_tension_model(p, t).flatten(),
+            gas_mass_fraction_model(p, t).flatten(),
+            gas_specific_heat_model(p, t).flatten(),
+            oil_specific_heat_model(p, t).flatten(),
+            gas_specific_enthalpy_model(p, t).flatten(),
+            oil_specific_enthalpy_model(p, t).flatten(),
+            gas_thermal_conductivity_model(p, t).flatten(),
+            oil_thermal_conductivity_model(p, t).flatten(),
+            temperature(p,h).flatten()
+        ]
+
+        names = [
+            "gas density",
+            "gas density derivative pressure",
+            "gas density derivative temperature",
+            "oil density",
+            "oil density derivative pressure",
+            "oil density derivative temperature",
+            "gas viscosity",
+            "oil viscosity",
+            "gas-oil surface tension",
+            "gas mass fraction",
+            "gas specific heat",
+            "oil specific heat",
+            "gas specific enthalpy",
+            "oil specific enthalpy",
+            "gas thermal conductivity",
+            "oil thermal conductivity",
+            "temperature"
+        ]
+
+        return PvtModelTableParametersDescription(
+            pressure_values=pressure_values,
+            temperature_values=enthalpy_values,
+            table_variables=data,
+            variable_names=names,
+            pressure_std=Scalar(1e5, "Pa"),
+            temperature_std=Scalar(15, "degC"),
+            gas_density_std=Scalar(rho_g_ref, "kg/m3"),
+            oil_density_std=Scalar(rho_l_ref, "kg/m3"),
+            water_density_std=Scalar(rho_w_ref, "kg/m3"),
+            gas_oil_ratio=Scalar(0, "sm3/sm3"),
+            gas_liquid_ratio=Scalar(0, "sm3/sm3"),
+            water_cut=Scalar(0, "-"),
+            total_water_fraction=Scalar(0, "-"),
+            number_of_phases=3 if has_water else 2,
+            table_type=constants.PVTTableType.PH.value
```

Comment:
Here the PH table type is specified and the `mode` of the `PvtModel` can be specified.